### ISSUE TITLE
[a11y][Input]: Voice over cannot read error message unless focus on the field

### ISSUE THREAD

**Issue by InfiniteXyy** [SAP/ui5-webcomponents#11493]
### Bug Description

The voiceOver cannot read the error message unless focus on the field (By default, the error for the entire page is not visible to the user.)

<img width="610" alt="Image" src="https://github.com/user-attachments/assets/2e6536de-5f4e-4ca4-be94-11c4d9a904c6" />

### Expected Behaviour

The Voice Over should be able to detect the error messages even user didn't focus on the field

### Steps to Reproduce

1. Go to example page https://sap.github.io/ui5-webcomponents/components/Input/
2. Open voice over
3. Hover on the field (not focus on it)
4. The voice over cannot read the error message

### Priority

None

### UI5 Web Components Version

2.10.0

### Browser

Chrome

### Operating System

macOS 15.4.1

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by NakataCode** [comment]
Hi @InfiniteXyy,

From what I see in the screenshot and description, the issue refers to a `<ui5-input>` in an error state, not a SelectDialog. According to the [Fiori accessibility guidelines](https://experience.sap.com/fiori-design-web/input-field/), error messages (value state text) are expected to be announced by screen readers like VoiceOver only when the input field is focused, not on hover. Hovering is not expected to trigger announcements for such messages.

As this aligns with the expected behavior, I’ll proceed with closing the issue.

Best regards,
Nikola

**Comment by InfiniteXyy** [comment]
> Hi [@InfiniteXyy](https://github.com/InfiniteXyy),
> 
> From what I see in the screenshot and description, the issue refers to a `<ui5-input>` in an error state, not a SelectDialog. According to the [Fiori accessibility guidelines](https://experience.sap.com/fiori-design-web/input-field/), error messages (value state text) are expected to be announced by screen readers like VoiceOver only when the input field is focused, not on hover. Hovering is not expected to trigger announcements for such messages.
> 
> As this aligns with the expected behavior, I’ll proceed with closing the issue.
> 
> Best regards, Nikola

Hi, sorry for the typo, the issue should be titled with ui5-input, not select dialog. And thank you for your detailed explanation.
